src/db/types.ts

```diff
@@ -1,7 +1,32 @@
 import { ObjectId } from "mongodb";
+export interface User {
+	_id?: ObjectId;
+	isBlind: boolean;
+	firstName: string;
+	lastName: string;
+	email: string;
+	reviews: Array<ObjectId> | Array<null>;
```

Comment:
wouldn't an empty array of ID's still be an `Array<ObjectID>`?

Reply:
I don't think ObjectID can be null but I'm not positive on that. Its possible that there are no reviews given, so I think instead I should have made this `Array<ObjectId> | null`.

Reply:
Should I change it to that? ^

Reply:
no I mean something like this

```
let emptyArray: Array<ObjectID> = [];
```

just because it's empty doesn't mean its null.

Reply:
Oh I totally see what you're saying now. That would work because we can initialize it to an empty array when the user is first created, but it wouldn't mean it's an array of nulls necessarily. I guess we can also initialize it to be null too (just null, not Array< null >) because how we initialize it is up to us. Is there anything specific you want me to do here? I'm thinking maybe null would be best since setting it to be Array would open the door to other issues since the primary data type is Array< ObjectID >. So what I mean is have it be 'Array < ObjectID > | null'.

Reply:
I think that if there are no reviews, it can just be an empty array. no need for null

Reply:
I think this should just be ` Array<ObjectId>` if it's empty then that type is still correct. "An empty array of ObjectId"